cinema/serializers.py

```diff
@@ -112,14 +112,17 @@
 
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
-        data = super(TicketSerializer, self).validate(attrs=attrs)
-        Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+        ticket = Ticket(
+            row=attrs["row"],
+            seat=attrs["seat"],
+            movie_session=attrs["movie_session"]
         )
-        return data
+        try:
+            ticket.clean()
+        except DjangoValidationError as e:
+            raise serializers.ValidationError({"detail": str(e)})
```

Comment:
You are raising a ValidationError here. However, it would be more helpful to the user if you could provide a more specific error message. Instead of simply converting the DjangoValidationError to a string, consider providing a custom error message that explains what went wrong.